repos/spack_repo/builtin/packages/ascent/package.py

```diff
@@ -761,25 +741,6 @@ def hostconfig(self):
         else:
             cfg.write("# occa not built by spack \n")
 
-        #######################
-        # Devil Ray
-        #######################
-        if spec.satisfies("+dray"):
-            cfg.write("# devil ray\n")
-            if self.spec.satisfies("@0.8.1:"):
-                cfg.write(cmake_cache_entry("ENABLE_DRAY", "ON"))
-                cfg.write(cmake_cache_entry("ENABLE_APCOMP", "ON"))
-            else:
-                cfg.write("# devil ray from spack \n")
-                cfg.write(cmake_cache_entry("DRAY_DIR", spec["dray"].prefix))
-        else:
-            if self.spec.satisfies("@0.8.1:"):
-                cfg.write("# devil ray\n")
-                cfg.write(cmake_cache_entry("ENABLE_DRAY", "OFF"))
-                cfg.write(cmake_cache_entry("ENABLE_APCOMP", "OFF"))
-            else:
-                cfg.write("# devil ray not build by spack\n")
```

Comment:
I'm not certain that the variant isn't still effective to control whether the `dray` portion of the source is built (can't tell from the packages). @cyrush can you confirm whether the variant should still be present absent supporting old external versions of dray?

Reply:
A while back, Devil Ray was moved to be developed inside of Ascent (separate repo was frozen). That said, it is still an option in Ascent - it can be turned on or off for a build. For newer ascent variant was used to select if it should be on or off for a given build.